lib/recurly/pager.php

```diff
@@ -8,23 +8,24 @@
  */
 abstract class Recurly_Pager extends Recurly_Base implements Iterator
 {
-  private $_etag;
```

Comment:
I'll leave myself a note to do an audit on how the other libs handle etags and see what we want to do. No need to deal with it now.